### ISSUE TITLE
Remove max-width in the css

### ISSUE THREAD

**Issue by FredM67** [emoncms/graph#132]
I cannot see any advantage to set a max width for any view. When displayed on a "big" monitor (for me 1920x1200), the view is displayed on about 66% of the width of the screen. Why?
When I remove this parameter at the beginning of the css file (or set it to a much bigger value), the view takes the whole window width.

Same apply for other views...

**Comment by glynhudson** [comment]
Agree, I don't see the need for the restriction.  The Feeds/Inputs pages can scale to any width.

@TrystanLea is there a reason for this restriction? 

![Emoncms - graph 192 - Google Chrome_442](https://user-images.githubusercontent.com/758844/121053676-83d85b80-c7b3-11eb-9abd-5cb8a4b84f9b.jpg)
![Emoncms - feed view - Google Chrome_443](https://user-images.githubusercontent.com/758844/121053682-85098880-c7b3-11eb-91f0-93d850683b0d.jpg)
